pkg/collector/corechecks/ebpf/probe/ebpfcheck/probe_test.go

```diff
@@ -252,12 +252,11 @@ func TestHashMapNumberOfEntries(t *testing.T) {
 
 func TestHashMapNumberOfEntriesNoExtraAllocations(t *testing.T) {
 	ebpftest.RequireKernelVersion(t, minimumKernelVersion)
-	entriesToTest := []uint32{10, 100, 1000, 10000}
+	entriesToTest := []uint32{24, 104, 1000, 10000} // We will divide by 8 in total to get some limits for the MultipleBatch test, ensure all numbers are divisible by 8
```

Comment:
nit: maybe woryh introducing a const and multiply it by factor for different batch sizes